docs/configuration/topic-routing.asciidoc

```diff
@@ -0,0 +1,55 @@
+= Topic Routing
+:awestruct-layout: doc
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+Debezium enables you to re-reoute the emitted change before the message reaches the converter using a 
+single message transformation, or https://kafka.apache.org/documentation/#connect_transforms[SMT]. 
+The SMT provided by Debezium enables you to rewrite the topic and the key according to a regular expression 
+and a replacement pattern, configurable per instance of Debezium.
+
+The implementation does not care about the sanity of the change, this is in the responsibility of the user.
+
+== Use-Cases
+
+=== Virtual Tables
+
+A logical table consists of one or more physical tables with the same table structure. A common use case is sharding. The
+two physical tables `db_shard1.my_table` and `db_shard2.my_table` together form one logical table.
```

Comment:
Maybe "A common use case is sharding, where for example two physical tables..."?